### ISSUE TITLE
Remove empty spartanization menu in problem

### ISSUE THREAD

**Issue by yossigil** [SpartanRefactoring/Spartanizer#184]
If you click on a problem, you see an empty spartanization menu

**Comment by OriRoth** [comment]
Any idea for problem menu options list? same as marker menu?

**Comment by yossigil** [comment]
Not at this time. I do not want to expand this. We have too many issues already, and I would like to get the GUI to be clean, even if it is not optimal

**Comment by OriRoth** [comment]
You removed this feature while cleaning `plugin.xml` (i.e. there is no spartanization menu in problem view). Do you want a replacement (consider #183) or can I close the issue?

**Comment by yossigil** [comment]
OOPS, I want this on. Sorry, there is a mysterious bug I am hunting day and night, the removal was one way to battle it.  Something regarding #226.

So please enable it. And make it identical to the main Refacor/Spartanization menu. If possible, by not cut and paste, but if ain't possible, by real cut and paste.

So: Yes

**Comment by OriRoth** [comment]
OK there is a spartanization menu in problem view. I managed to limit the menu contribution only for the problem view (and not for projects, for instance). BUT it appears not only for tips, but for other warnings as well (bug!). I tried to deal with the problem, but it remains **unresolved** for now.
Some relevant links:
* http://stackoverflow.com/questions/11852497/add-popup-menu-to-eclipse-problems-view
* http://help.eclipse.org/luna/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fui%2Fmenus%2FMenuUtil.html (not sure about that)
* http://stackoverflow.com/questions/7049670/how-to-limit-visibility-of-a-popup-menu-only-to-a-certain-project-type (i tried to implement a PropertyTester, but it was not recognized by eclipse for some reason)
It may become a new issue.
